graphql_compiler/schema_merging/rename_schema.py

```diff
@@ -0,0 +1,232 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+
+from graphql import build_ast_schema, parse
+from graphql.language.visitor import Visitor, visit
+
+from .utils import SchemaError, get_schema_data
+
+
+RenamedSchema = namedtuple(
```

Comment:
I am a bit concerned about user confusion between this type and the built-in `GraphQLSchema` type. It would not be unreasonable for an unfamiliar user to assume that, since both objects claim to be schemas, they would be freely interchangeable and mutually compatible. This is of course not the case, so we risk having bugs here down the line.

I unfortunately don't have a concrete suggestion for a better name, but I'm happy to sit down and brainstorm one together if you'd like.